interpreter/interpretable.go

```diff
@@ -124,7 +126,14 @@ func (test *evalTestOnly) Eval(ctx Activation) ref.Val {
 		return container.Contains(test.field)
 	}
 	return types.ValOrErr(obj, "invalid type for field selection.")
+}
 
+// Cost implements the Coster interface method.
+func (test *evalTestOnly) Cost() (min, max int64) {
+	min, max = estimateCost(test.op)
```

Comment:
Make sure you comment on why the cost for test only is at least `1` when the cost fo field accesses is `0`

Reply:
done